sdk/storage_queues/src/operations/get_messages.rs

```diff
@@ -40,8 +40,8 @@ pub struct GetMessagesResponse {
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct Message {
-    message_id: String,
-    pop_receipt: String,
+    pub message_id: String,
+    pub pop_receipt: String,
```

Comment:
Actually now I'm confused why you would want to use these fields directly instead of using the `pop_receipt` method below. Can you clarify? If there's a good answer, we should document it on the `pop_receipt` method.

Reply:
Both are acceptable, but before this PR, even PopReceipt didn't offer access as well. Now I do worry about cloning the strings when we need this access.

Although I think it's fairly cheap to clone both of them, but I think it has a potential to be costly.

To avoid mutations, we can add methods to expose these by reference, like the PopReceipt.